viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -398,7 +455,7 @@ import { mixin as clickaway } from 'vue-clickaway';
 import Utils from '../utils/utils';
 
 // d3 force directed graph vars/functions ---------------------------------- */
-let colors, foregroundColor;
```

Comment:
the changes here are pretty much just to visually distinguish the types of nodes (baseline, new, both)